EMS/core-bundle/src/Controller/Api/Form/SubmissionController.php

```diff
@@ -52,4 +54,32 @@ public function detail(Request $request, string $submissionId): JsonResponse
 
         return new JsonResponse($submission);
     }
+
+    public function submissionFile(string $submissionId, string $submissionFileId): JsonResponse|StreamedResponse
+    {
+        if (null === $submissionFile = $this->formSubmissionService->findFile($submissionId, $submissionFileId)) {
+            return new JsonResponse([], Response::HTTP_NOT_FOUND);
+        }
+
+        $response = new StreamedResponse(function () use ($submissionFile) {
+            if (null === $fileStream = $submissionFile->getFile()) {
+                exit;
+            }
+
+            while (!\feof($fileStream)) {
+                echo \fread($fileStream, 1);
```

Comment:
1 byte at the time?

Reply:
use default_chunk_size constant from the file helper class